src/core/logical_layers/renderers/BivariateRenderer.tsx

```diff
@@ -224,30 +224,31 @@ export class BivariateRenderer extends LogicalLayerDefaultRenderer {
             }
           : undefined;
 
-      currentTooltipAtom.setCurrentTooltip.dispatch({
-        initiatorId: TOOLTIP_ID,
-        popup: (
-          <MapHexTooltip
-            cellLabel={cells[cellIndex].label}
-            cellIndex={cellIndex}
-            axis={legend.axis}
-            values={valuesByQuotients}
-            hexagonColor={rgba}
-          />
-        ),
-        popupClasses: { popupContent: popupContentRoot },
-        position: {
-          x: ev.originalEvent.clientX,
-          y: ev.originalEvent.clientY,
-        },
-        onClose(_e, close) {
-          close();
-          cleanSelection();
-        },
-        onOuterClick(_e, close) {
-          close();
-          cleanSelection();
-        },
+      const popupNode = document.createElement('div');
+      createRoot(popupNode).render(
+        <MapHexTooltip
+          cellLabel={cells[cellIndex].label}
+          cellIndex={cellIndex}
+          axis={legend.axis}
+          values={valuesByQuotients}
+          hexagonColor={rgba}
+        />,
+      );
+
+      if (this._popup) cleanPopup();
+      this._popup = new Popup({
+        closeOnClick: true,
+        className: 'bivariateHexagonPopupContentRoot',
```

Comment:
this className is very specific - so it's weird we add it to global css
can we use it in our usual flow with `import s from 'some.module.css'` and `className={s.bivariateHexPopup}`?

Reply:
yeah, i tried with usual flow, but it didn't work
it seems because only top classname is from module.css and all others are not

Reply:
aah, I just need to use :global scope